experiment/measurer/measure_manager.py

```diff
@@ -70,21 +72,29 @@ def measure_main(experiment_config):
     initialize_logs()
     logger.info('Start measuring.')
 
+    # Create data frame container for segment and function coverage info.
+    experiment_specific_df_container = (
```

Comment:
This name is super vague. Let's rename it once we've renamed DataFrameContainer.

Reply:
Change `experiment_specific_df_container` to `experiment_coverage_data_container`